### ISSUE TITLE
Add Value Curve to Single Strand effect for Chase Type

### ISSUE THREAD

**Issue by idleup** [smeighan/xLights#585]
My goal is to control movement with a value curve so I can make the chase start at 0% and move to 50% then back to 25% and then to 100% at the end of the effect. Almost like a progressive bouncing fill.

**Comment by kppankratz** [comment]
Looks like Keith is slowly adding the value curve to other effects (4 more in the last two days/since the last release).  Just a matter of time until he gets to the single strand.

**Comment by kppankratz** [comment]
This will be in the next release.

**Comment by AzGilrock** [comment]
I was about to close this thinking it was done but I released the 3 value curves added were not related to this request.  Currently "Chase Type" is a combo-box drop down to control the movement so there is still no way to use a value curve to control the movement of the chase across the element.  So it probably need to be switched like I did for Fill where you just draw it at a position and let the curve control the position.  Maybe have a way to quickset the existing chase types into curves.  I'm sure this one may be more difficult to actually implement in the effect rendering code.  If the position of the chase quickly jumps to another position how is the tail drawn?  Reading the description above again it almost sounds like the new Fill satisfies that request.  Fill was easier because I always filled over to the position.  A chase with a shorter tail is harder to figure out how to render using a value curve.
